Tests/Form/Type/DateTimePickerTypeTest.php

```diff
@@ -24,7 +24,7 @@ class DateTimePickerTypeTest extends \PHPUnit_Framework_TestCase
 {
     public function testGetName()
     {
-        $type = new DateTimePickerType(new MomentFormatConverter());
+        $type = new DateTimePickerType(new MomentFormatConverter(), $this->getMock('Symfony\Component\Translation\TranslatorInterface'));
```

Comment:
Not really.

Make one test with the old constructor signature should just to match the condition statement you introduced.

Call it `testLegacyConstructor` to make phpunit happy with it.

See: https://github.com/symfony/phpunit-bridge